Package.swift

```diff
@@ -20,35 +20,34 @@ import PackageDescription
 let package = Package(
   name: "RecaptchaEnterprise",
   platforms: [
-    .iOS(.v11)
+    .iOS(.v12)
   ],
   products: [
     // Products define the executables and libraries a package produces, and make them visible to other packages.
     .library(
       name: "RecaptchaEnterprise",
-      targets: ["recaptcha-enterprise"]
-    )
+      targets: ["RecaptchaEnterprise"]
+    ),
   ],
   dependencies: [
     .package(
       url: "https://github.com/google/interop-ios-for-google-sdks.git",
-      "100.0.0"..<"101.0.0"
+      "101.0.0"..<"102.0.0"
     )
   ],
   targets: [
+    .binaryTarget(
+      name: "RecaptchaEnterpriseSDK",
+     url:
+       "https://dl.google.com/recaptchaenterprise/v18.7.0-beta01/RecaptchaEnterpriseSDK_iOS_xcframework/recaptcha-sdk-xcframework.xcframework.zip",
+     checksum: "170769f1ba86a0ce422a80f127cd4ffca999b360ad6fc3fcc394eb47e0d97d06"
+    ),
     .target(
-      name: "recaptcha-enterprise",
+      name: "RecaptchaEnterprise",
       dependencies: [
-        "RecaptchaEnterprise",
+        .target(name: "RecaptchaEnterpriseSDK"),
```

Comment:
Instead of depending on `RecaptchaEnterpriseSDK` directly, create a new target for the redirect header, have that new target depend on `RecaptchaEnterpriseSDK`, and depend on that new target here.

Reply:
Thanks @ncooke3, changes made using your suggestions and seems to be working from the tests I made.